tuxemon/core/prepare.py

```diff
@@ -180,3 +177,16 @@ def init():
     # Initialize PyGame and our screen surface.
     if PLATFORM == 'pygame':
         pygame_init()
+
+# Fetches a resource file
+def fetch(*args):
+    path = '/'.join(i for i in args if i is not None)
```

Comment:
use `os.path.join`

Reply:
I think I tried and it didn't work, will attempt this again and see what fails if anything does.

Reply:
The change fails with the following error:

`AttributeError: 'generator' object has no attribute 'startswith'`

Reply:
please post the code that raised that traceback

Reply:
`path = os.path.join(i for i in args if i is not None)`

Reply:
there should never be a `None` passed in.  there should always be an argument, so why not
```path = os.path.join(*args)```?

Reply:
Never mind, a much easier solution was available all along:

`path = os.path.join(*args)`